libs/execution/src/lib/blocks/block-executor.ts

```diff
@@ -52,6 +54,15 @@ export abstract class AbstractBlockExecutor<I extends IOType, O extends IOType>
       return;
     }
 
+    const currentNode = context.getCurrentNode();
+    assert(isBlockDefinition(currentNode));
+    const isBlockTargeted =
```

Comment:
This should be a function somewhere (on the DebugLogVisitor (?)) so it is easier to change in the future.